File: Siddhant Gedekar/Java_Codes/Practice Set 1/digitSum.java
public class digitSum {
    public static void main(String [] args){
        int digit = 1234;
        int res = 0;
        for(int i=0;i<4;i++){
            int num = digit%10;
            digit /= 10;
            res = num + res;
        }
        System.out.println("Sum of digits is: "+res);
    }
}

File: Siddhant Gedekar/C_Cpp_Codes/Practice Set 3/digitSum.c
#include<stdio.h>

int main(){
    int digit;
    int num=0,sum=0;
    printf("Enter any digit number: ");
    scanf("%d",&digit);
    while(digit!=0){
        sum++;
        num=digit%10 + num;
        digit/=10;
    }
    printf("%d",num);
    return 0;
}